src/latest_release_note.ts

```diff
@@ -1,21 +1,43 @@
-export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🐍 (Mar. 31, 2020)</h1>
+export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes ⌨️🐍💞 (April 15, 2020)</h1>
 <p>
-    We're back with a small hotfix!</p>
-<p>
-    <h2>Fixes:</h2>
-    <ul>
-        <li>Fixed serial port selection for serial monitor.</li>
-        <li>Fixed launch issue on Ubuntu.</li>
-    </ul>
-
-
-</p>
-<p>
-    <h2>Changes:</h2>
-    <li>Due to necessary serial monitor fixes, <b>the extension is now only supported on VS Code v1.43+</b></li>
-
-</p>
-<br>
-<p><b>Stay healthy while you hack 🥰🌼,</b><br>
-    &nbsp&nbsp&nbsp&nbsp&nbsp <b><i>- The Device Simulator Express Team</i></b></p>`;
+    We're unveiling a new addition to our DSX family of microcontroller simulators! Please welcome our <b>Adafruit CLUE
+        simulator</b> 💕🔍.<br>
+    This change is <b>hidden</b> under our preview flag by default. <a href="https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/README.md#how-to-enable-preview-flag">See here</a> to learn how to enable
+    preview mode!
+    <p>
+        <h2>Changes</h2>
+        <ul>
+            <li>Added the support for the <b>Adafruit CLUE</b>! This is hidden behind a preview flag by default.
+                <ul>
+                    <li>
+                        Added CLUE Simulator, featuring simulation for the following:
+                        <ul>
+                            <li>240x240 color display.</li>
+                            <li>Sensors (for temperature, light, color, acceleration, humidity, pressure, proximity,
+                                gestures, gyro, and magnetic field).</li>
+                        </ul>
+                    </li>
+                    <li>Implemented deploy-to-device for CLUE.</li>
+                    <li>Created CLUE code debugger.</li>
+                </ul>
+            </li>
+            <li>Command palette only shows actions (ie: Open Simulator, Deploy to Device) once instead of per device.
+            </li>
+            <li>New "Getting Started" page containing CPX, micro:bit, and CLUE example code.</li>
+            <li>Support for gesture simulation on the micro:bit. The simulated gestures are: up, down, left, right, face
+                up,
+                face down, freefall, 3g, 6g, 8g, and shake.</li>
+            <li>Tab UI now has callout for "about" information on each sensor/input option.</li>
+            <li>The CircuitPython Neopixel and Adafruit_Fancyled libraries can now be used with the Adafruit CPX without
+                importing the CPX library. <a
+                    href="https://circuitpython.readthedocs.io/projects/fancyled/en/latest/examples.html">See here</a>
+                for an example.</li>
+        </ul>
+        <h2>Fixes:</h2>
+        <ul>
+            <li>Fixed issue with sensors on micro:bit debugger.</li>
+        </ul>
+    </p>
+    <p><b>Keep being a coding champ 🥳🏆🙌,</b><br>
```

Comment:
Perhaps replace the sick emoji with something more positive?

Reply:
I think the 🥳 emoji it a party emoji!! I can replace it though ^^

Reply:
Super sorry, I should have zoomed in!
Up to you!